task/07-yield-tasks.js

```diff
@@ -46,8 +59,27 @@ function* get99BottlesOfBeer() {
  * @return {Iterable.<number>}
  *
  */
+
+
 function* getFibonacciSequence() {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+	let x = 0,
+		 y = 1;
+	while (x >= 0) {
+		if (x == 0) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

Reply:
fixed